# reference points
reference_point:
  filepath: /home/pilyang/wps/logs/rp_skirk2.csv
  distance: 1.5

# name of topic to subscribe and publish
topic:
  scan: /scan
  odom: /odom
  nav: /drive


# configuration values for driving algorithm
driving:
  speed_max: 3.5
  speed_min: 1.5
  robot_scale: 0.2
  threshold: 6.0
  gap_size: 4
  filter_scale: 1.1
  gap_theta_gain: 10.0
  ref_theta_gain: 1.0
